### DISCUSSION TITLE
Problems installing the integration

### DISCUSSION THREAD

**Discussion by stephanlarsson** [ekutner/home-connect-hass#146] [Community Support]
Hi I've been trying to install this integration for several weeks now and can't understand where I'm going wrong. I manage to start the Oath process and sign in using the Single-ID, but when I get back to the HA instance I get a blank error message. Can anyone help me figure out why I'm seeing this?
<img width="685" alt="Screenshot 2023-01-03 at 15 51 53" src="https://user-images.githubusercontent.com/120938592/210505341-4aa5a55e-367c-4bff-93b6-5125b83e9c8b.png">

**Comment by ekutner** [comment]
It is always a good idea to start by searching for information. There is literally the exact issue two items down in the discussions board.

**Reply by stephanlarsson** [reply]
Hi,

Well aware. Like I said, been trying to solve this issue for several weeks now. That included reading about other peoples issues here. Unfortunately that hasn’t been able to help me with my issue of the blank error message. Hence my question.

Do you have any what might be the issue?

**Reply by stephanlarsson** [reply]
Done some more digging. I think it may be related to the fact that I cannot reach my home assistant instance by using the homeassistant.local:8123 link in my browser... Now to figure out how to fix that...

**Comment by ekutner** [comment]
It shouldn't matter how you access your HA instance, as long as you do it directly, not through Nabu Casa, and are consistent about it, so that you add the integration and initiate the HC authentication flow from the same domain you configure in https://my.home-assistant.io/
